docs/source/getting_started/detailed_tutorial.md

```diff
@@ -227,6 +226,13 @@ ChatCompletionResponse(
 Alternatively, you can run inference using the Llama Stack client SDK.
 
 ### i. Create the Script
+
```

Comment:
Do you mind taking this out actually? I have a more complete update in another PR

Reply:
see here: https://github.com/meta-llama/llama-stack/pull/1935